go.mod

```diff
@@ -21,11 +21,13 @@ require (
 	golang.org/x/mod v0.16.0
 	golang.org/x/net v0.22.0
 	google.golang.org/grpc v1.62.1
+	gopkg.in/yaml.v2 v2.4.0
 	k8s.io/api v0.29.3
 	k8s.io/apiextensions-apiserver v0.29.3
 	k8s.io/apimachinery v0.29.3
 	k8s.io/client-go v0.29.3
 	k8s.io/klog/v2 v2.120.1
+	k8s.io/kubectl v0.29.2
```

Comment:
```suggestion
	k8s.io/kubectl v0.29.3
```

Reply:
Also a question: Does it make sense to make `tests/e2e`  a go submodule to ensure that we don't have unneeded dependencies in the root go.mod file?

Reply:
yeah probably, I am also thinking on moving most of `common` folder to `NVIDIA/k8s-test-infra` so we can share this logic across all repos and avoid code duplication